bindata.go

```diff
@@ -86,7 +86,7 @@ func bashenvBashBash() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "bashenv/bash.bash", size: 557, mode: os.FileMode(420), modTime: time.Unix(1485014092, 0)}
+	info := bindataFileInfo{name: "bashenv/bash.bash", size: 557, mode: os.FileMode(420), modTime: time.Unix(1492033206, 0)}
```

Comment:
I don't fully understand this change, the file `bashenv/bash.bash` didn't change ...

Reply:
Oh is it because after forking the timestamp of that file's _creation_ changes, but still the new timestamp doesn't seem right, so I'm not sure

Reply:
I don't understand it either but the convention with `bindata.go` seems to be to treat it as a blob and commit whatever the toolchain spits out.

Reply:
OK I see, thank you!